Chapter1/Zad4.py

```diff
@@ -0,0 +1,25 @@
+'''
+Created on 28 sty 2018
+
+@author: agata.wiewiora
+
+This program that reads an integer number and prints its 
+previous and next numbers.
+
+'''
+
+
+try:
+    # Read an integer:
+    integer_ = raw_input("Please, provide sample integer: ")
```

Comment:
PEP zaleca nazywać zmienne małymi literami z kolejnymi wyrazami odseparowanymi underscorem, https://www.python.org/dev/peps/pep-0008/#function-and-variable-names nie rozumiem tego sufiksu

Reply:
dodatkowo integer_ wcale nie jest na tym etapie intem tylko stringiem przez zastosowanie raw_input, więc nazwa jest trochę myląca